cmd/cmd.go

```diff
@@ -33,3 +51,221 @@ const msgMissingMetadata = `
     Please see https://exercism.io/cli-v1-to-v2 for instructions on how to fix it.
 
 `
+
+type downloadContext struct {
+	usrCfg  *viper.Viper
+	uuid    string
+	slug    string
+	track   string
+	team    string
+	payload *downloadPayload
+}
+
+func newDownload(ctx *downloadContext) error {
```

Comment:
I'm not sure `newDownload` is the right name here. Generally if I see a function `newDownload` I would expect it to return a `Download` value or a pointer to a download value. In this case it _performs_ a download, right?